code/modules/power/cable.dm

```diff
@@ -115,14 +115,8 @@ By design, d1 is the smallest direction and d2 is the highest
 	else
 		icon_state = "[d1]-[d2]"
 
-
-// Items usable on a cable :
-//   - Wirecutters : cut it duh !
-//   - Cable coil : merge cables
-//   - Multitool : get the power currently passing through the cable
-//
-/obj/structure/cable/attackby(obj/item/W, mob/user, params)
-	var/turf/T = src.loc
+/obj/structure/cable/proc/fuckcodestandards(obj/item/W, mob/user, params)

```

Comment:
Yeah, I knew I was gonna be asked that. That was just named that because I hated having to move stuff into new procs